src/api/fileops/fileset.ts

```diff
@@ -0,0 +1,45 @@
+import { Inject, Injectable } from "injection-js";
+import { Observable } from 'rxjs';
+import { IResource } from '../core/resource';
+import { FilesetName } from '../fileops/fileops.interfaces';
+import { FileUploader } from '../fileops/fileUploader';
+import { FilesetMultipart, IFileUploadStatus } from './fileops.interfaces';
+
+/**
+ * Fileset object is used for manipulate files
```

Comment:
`...used TO manipulate files`

Reply:
@marlosin  "object is used to manipulate files" means that this object has a habit to manipulate files, which sounds pretty weird 😂 
"Used for" means "employed as a tool" - which is perfectly fit here

Reply:
but, there is another mistake, should've been 'manipulating'